INTEGRATION_TESTING.md

```diff
@@ -67,6 +67,13 @@ subdirectory.  For example:
 ...
 ```
 
+The `prereq` approach solves the problem where the top-level module has a
+count/for_each dependency on the supporting resources being created in
+the test.  It creates two terraform state files, one for the prereq and
+one for the test config. The approach here is just a way of separating
```

Comment:
The last sentence is not relevant to the prereq approach. That was specific to the ldap module.

Reply:
ok